internal/db/migrations/1_create_table_images.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS composes;
```

Comment:
What's the intended use of this?

Reply:
Completeness, for every migration up there should be a migration down. In case we ever reach a situation where we wanna migrate down, it's very unlikely we'll reach this migration ever, but idk completeness :)